app/views/RoomsListView/index.js

```diff
@@ -601,6 +601,7 @@ class RoomsListView extends React.Component {
 			<RoomItem
 				alert={item.alert}
 				unread={item.unread}
+				hideUnreadStatus={item.hideUnreadStatus}
```

Comment:
You need to add this value to `Subscription` model on Database or this value not persist if it is offline..